src/components/Filters/Filters.scss

```diff
@@ -1,6 +1,6 @@
 @import '../../styles/common';
 
-$list-filters-header-height: rem(56px);
+$list-filters-header-height: top-bar-height();
```

Comment:
This is the same value, but this makes it clearer that they're related, especially given the mentions of `--p-top-bar-height` below